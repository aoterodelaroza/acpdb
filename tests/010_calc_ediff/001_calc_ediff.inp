## check: 001_calc_ediff.out -a1e-10
## delete: 001_calc_ediff.db
## labels: regression quick

system rm -f 001_calc_ediff.db
connect 001_calc_ediff.db

insert method test
end

insert litref bibtex ../zz_source/bsip2.bib
insert set s22
 description A description.
 litrefs bsip1 boys1970
 din ../zz_source/s22.din
 directory ../zz_source/s22_xyz
end
insert property ediff.
 property_type ENERGY
 set s22
end
## for writing the input files
# write
#  template template.gjf
#  set s22
# end
insert calc
 property_type ENERGY
 file ../zz_source/s22.sto-3g.energies
 method test
end
calc_ediff
print evaluation
