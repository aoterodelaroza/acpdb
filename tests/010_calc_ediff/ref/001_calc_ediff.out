%% verbose
%% system rm -f 001_calc_ediff.db
* SYSTEM: rm -f 001_calc_ediff.db

%% connect 001_calc_ediff.db
* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 001_calc_ediff.db
Creating skeleton database 

%% insert method test
* INSERT: insert data into the database (METHOD)
# INSERT METHOD test

%% insert litref bibtex ../zz_source/bsip2.bib
* INSERT: insert data into the database (LITREF)
# INSERT LITREF bsip1
# INSERT LITREF boys1970
# INSERT LITREF s30l
# INSERT LITREF s12lb
# INSERT LITREF s12la
# INSERT LITREF risthaus2013
# INSERT LITREF xdmlarge
# INSERT LITREF johnson2009
# INSERT LITREF burns2017
# INSERT LITREF parker2015
# INSERT LITREF smith2016
# INSERT LITREF thanthiriwatte2010
# INSERT LITREF marshall2011
# INSERT LITREF burns2011
# INSERT LITREF bauza2013
# INSERT LITREF johnson2003
# INSERT LITREF wilke2009
# INSERT LITREF zhao2005
# INSERT LITREF zhao2005b
# INSERT LITREF grimme2007
# INSERT LITREF parthiban2001
# INSERT LITREF karton2011
# INSERT LITREF kozuch2013
# INSERT LITREF binkley1980
# INSERT LITREF gordon1982
# INSERT LITREF pietro1982
# INSERT LITREF dobbs1987a
# INSERT LITREF dobbs1987b
# INSERT LITREF dithcfield1971
# INSERT LITREF hehre1972
# INSERT LITREF hariharan1973
# INSERT LITREF hariharan1974
# INSERT LITREF gordon1980
# INSERT LITREF francl1982
# INSERT LITREF weigend2005
# INSERT LITREF weigend2006
# INSERT LITREF sto3g
# INSERT LITREF mini
# INSERT LITREF jensen1
# INSERT LITREF jensen2
# INSERT LITREF jensen3
# INSERT LITREF jensen4
# INSERT LITREF b3
# INSERT LITREF lyp
# INSERT LITREF dunning1
# INSERT LITREF dunning2
# INSERT LITREF grimme2010
# INSERT LITREF xdmbasis
# INSERT LITREF witte2016
# INSERT LITREF davidson1986
# INSERT LITREF andzelm1984
# INSERT LITREF kruse2012
# INSERT LITREF hbs66
# INSERT LITREF halogen
# INSERT LITREF dcacp
# INSERT LITREF qcp
# INSERT LITREF hfacp
# INSERT LITREF lasso
# INSERT LITREF osborne2000
# INSERT LITREF wb97xd1
# INSERT LITREF wb97xd2
# INSERT LITREF pepconf
# INSERT LITREF xdmhybrid
# INSERT LITREF xdm
# INSERT LITREF lcwpbe1
# INSERT LITREF lcwpbe2
# INSERT LITREF kalescky2014
# INSERT LITREF nbody1
# INSERT LITREF wateracp
# INSERT LITREF hf-3c
# INSERT LITREF kulik2016
# INSERT LITREF sure2015
# INSERT LITREF goerigk2013
# INSERT LITREF galano2006
# INSERT LITREF jensen2010
# INSERT LITREF hehre1970
# INSERT LITREF curtiss1999
# INSERT LITREF dill1975
# INSERT LITREF clark1983
# INSERT LITREF frisch1984
# INSERT LITREF rassolov1998
# INSERT LITREF mackie2011
# INSERT LITREF burns2013
# INSERT LITREF rezac2011
# INSERT LITREF dilabio2008
# INSERT LITREF mackie2008
# INSERT LITREF mackie2010
# INSERT LITREF torres2012
# INSERT LITREF dilabio2014
# INSERT LITREF vansanten2015
# INSERT LITREF goerigk2011
# INSERT LITREF aorss3
# INSERT LITREF aorss1
# INSERT LITREF aspirin
# INSERT LITREF clathrates
# INSERT LITREF aupi2
# INSERT LITREF pyricarbamate
# INSERT LITREF tmcomplex
# INSERT LITREF wallach
# INSERT LITREF junctions
# INSERT LITREF hb-piperidine
# INSERT LITREF 2nitrile
# INSERT LITREF chpi-pdxanthate
# INSERT LITREF digold
# INSERT LITREF abo3
# INSERT LITREF bipodal-carbamate
# INSERT LITREF oscil-friction
# INSERT LITREF tibond
# INSERT LITREF clathrate-hpr
# INSERT LITREF epxars-fepxas
# INSERT LITREF graphitestress-theo
# INSERT LITREF agtao3surf
# INSERT LITREF xdmchiral
# INSERT LITREF graphenemesa
# INSERT LITREF anales
# INSERT LITREF electrides
# INSERT LITREF xdmgold
# INSERT LITREF kevan
# INSERT LITREF burrito
# INSERT LITREF step
# INSERT LITREF agtao3
# INSERT LITREF xdmc9
# INSERT LITREF kaolin
# INSERT LITREF xdmbench
# INSERT LITREF ncisolids
# INSERT LITREF graphitestress
# INSERT LITREF xdmsolids
# INSERT LITREF energy-correction
# INSERT LITREF cij-partition
# INSERT LITREF supramolecular
# INSERT LITREF qhamgo
# INSERT LITREF ctc-gibbs
# INSERT LITREF gibbs2b
# INSERT LITREF gibbs2a
# INSERT LITREF sbasga2
# INSERT LITREF sbasga2-sbpga2
# INSERT LITREF aggase2
# INSERT LITREF cdal2se4
# INSERT LITREF lapla
# INSERT LITREF qtree
# INSERT LITREF variational
# INSERT LITREF sbpga2
# INSERT LITREF critic
# INSERT LITREF runwien
# INSERT LITREF b88
# INSERT LITREF s225
# INSERT LITREF s22
# INSERT LITREF s66
# INSERT LITREF s668
# INSERT LITREF g3
# INSERT LITREF s22b
# INSERT LITREF s66b
# INSERT LITREF kb49b
# INSERT LITREF kb49a
# INSERT LITREF p26
# INSERT LITREF l7
# INSERT LITREF hsg0
# INSERT LITREF ionichb
# INSERT LITREF gmtkn55
# INSERT LITREF gmtkn30
# INSERT LITREF bsr36
# INSERT LITREF pconf
# INSERT LITREF sconf
# INSERT LITREF water
# INSERT LITREF x40
# INSERT LITREF s8
# INSERT LITREF bde03
# INSERT LITREF aconf
# INSERT LITREF bhperi1
# INSERT LITREF bhperi2
# INSERT LITREF bhperi3
# INSERT LITREF bhperi4
# INSERT LITREF darc
# INSERT LITREF idisp
# INSERT LITREF s668b
# INSERT LITREF adim6
# INSERT LITREF a24
# INSERT LITREF karlsruhe1
# INSERT LITREF karlsruhe2
# INSERT LITREF pople1
# INSERT LITREF pople2
# INSERT LITREF pople3
# INSERT LITREF pople4
# INSERT LITREF pople5
# INSERT LITREF pople6
# INSERT LITREF pople7
# INSERT LITREF pople8
# INSERT LITREF bonifacic1974
# INSERT LITREF brauer2014
# INSERT LITREF clementi1982
# INSERT LITREF stevens1984
# INSERT LITREF topiol1980
# INSERT LITREF topiol1981
# INSERT LITREF duijneveldt1994
# INSERT LITREF davidson1994
# INSERT LITREF xdmpoly
# INSERT LITREF critic2
# INSERT LITREF kahn1972
# INSERT LITREF kahn1976
# INSERT LITREF haynes2006
# INSERT LITREF faver2011
# INSERT LITREF faver2012
# INSERT LITREF jensen2013
# INSERT LITREF goerigk2014
# INSERT LITREF cao2011
# INSERT LITREF dolg2011
# INSERT LITREF kulik2016b
# INSERT LITREF mentel2013
# INSERT LITREF mao2016
# INSERT LITREF deng2011
# INSERT LITREF gutowski1993
# INSERT LITREF xdmsurface
# INSERT LITREF carter2014
# INSERT LITREF linear1
# INSERT LITREF linear2
# INSERT LITREF linear3
# INSERT LITREF spackman2001
# INSERT LITREF linear4
# INSERT LITREF whitten1973
# INSERT LITREF dunlap1979
# INSERT LITREF feyereisen1993
# INSERT LITREF vahtras1993
# INSERT LITREF weigend2002
# INSERT LITREF sodt2006
# INSERT LITREF gpu1
# INSERT LITREF gpu2
# INSERT LITREF gpu3
# INSERT LITREF liechtenstein1995
# INSERT LITREF dudarev1998
# INSERT LITREF kabsch1976
# INSERT LITREF kaminsky2007
# INSERT LITREF maier2015
# INSERT LITREF hornak2006
# INSERT LITREF bsip2

%% insert set s22
* INSERT: insert data into the database (SET)
# INSERT SET s22
# INSERT STRUCTURE s22.nh3_nh3
# INSERT STRUCTURE s22.nh3_nh3_1
# INSERT STRUCTURE s22.nh3_nh3_2
# INSERT PROPERTY s22.nh3_nh3
# INSERT STRUCTURE s22.h2o_h2o
# INSERT STRUCTURE s22.h2o_h2o_1
# INSERT STRUCTURE s22.h2o_h2o_2
# INSERT PROPERTY s22.h2o_h2o
# INSERT STRUCTURE s22.h2co2_h2co2
# INSERT STRUCTURE s22.h2co2_h2co2_1
# INSERT STRUCTURE s22.h2co2_h2co2_2
# INSERT PROPERTY s22.h2co2_h2co2
# INSERT STRUCTURE s22.formamide_formamide
# INSERT STRUCTURE s22.formamide_formamide_1
# INSERT STRUCTURE s22.formamide_formamide_2
# INSERT PROPERTY s22.formamide_formamide
# INSERT STRUCTURE s22.uracil_uracil_hb
# INSERT STRUCTURE s22.uracil_uracil_hb_1
# INSERT STRUCTURE s22.uracil_uracil_hb_2
# INSERT PROPERTY s22.uracil_uracil_hb
# INSERT STRUCTURE s22.pyridoxine_aminopyridine
# INSERT STRUCTURE s22.pyridoxine_aminopyridine_1
# INSERT STRUCTURE s22.pyridoxine_aminopyridine_2
# INSERT PROPERTY s22.pyridoxine_aminopyridine
# INSERT STRUCTURE s22.adenine_thymine_wcc1
# INSERT STRUCTURE s22.adenine_thymine_wcc1_1
# INSERT STRUCTURE s22.adenine_thymine_wcc1_2
# INSERT PROPERTY s22.adenine_thymine_wcc1
# INSERT STRUCTURE s22.ch4_ch4
# INSERT STRUCTURE s22.ch4_ch4_1
# INSERT STRUCTURE s22.ch4_ch4_2
# INSERT PROPERTY s22.ch4_ch4
# INSERT STRUCTURE s22.c2h4_c2h4
# INSERT STRUCTURE s22.c2h4_c2h4_1
# INSERT STRUCTURE s22.c2h4_c2h4_2
# INSERT PROPERTY s22.c2h4_c2h4
# INSERT STRUCTURE s22.c6h6_ch4
# INSERT STRUCTURE s22.c6h6_ch4_1
# INSERT STRUCTURE s22.c6h6_ch4_2
# INSERT PROPERTY s22.c6h6_ch4
# INSERT STRUCTURE s22.c6h6_c6h6_pd
# INSERT STRUCTURE s22.c6h6_c6h6_pd_1
# INSERT STRUCTURE s22.c6h6_c6h6_pd_2
# INSERT PROPERTY s22.c6h6_c6h6_pd
# INSERT STRUCTURE s22.pyrazine_pyrazine
# INSERT STRUCTURE s22.pyrazine_pyrazine_1
# INSERT STRUCTURE s22.pyrazine_pyrazine_2
# INSERT PROPERTY s22.pyrazine_pyrazine
# INSERT STRUCTURE s22.uracil_uracil_stack
# INSERT STRUCTURE s22.uracil_uracil_stack_1
# INSERT STRUCTURE s22.uracil_uracil_stack_2
# INSERT PROPERTY s22.uracil_uracil_stack
# INSERT STRUCTURE s22.indole_c6h6_stack
# INSERT STRUCTURE s22.indole_c6h6_stack_1
# INSERT STRUCTURE s22.indole_c6h6_stack_2
# INSERT PROPERTY s22.indole_c6h6_stack
# INSERT STRUCTURE s22.adenine_thymine_stack
# INSERT STRUCTURE s22.adenine_thymine_stack_1
# INSERT STRUCTURE s22.adenine_thymine_stack_2
# INSERT PROPERTY s22.adenine_thymine_stack
# INSERT STRUCTURE s22.c2h4_c2h2
# INSERT STRUCTURE s22.c2h4_c2h2_1
# INSERT STRUCTURE s22.c2h4_c2h2_2
# INSERT PROPERTY s22.c2h4_c2h2
# INSERT STRUCTURE s22.c6h6_h2o
# INSERT STRUCTURE s22.c6h6_h2o_1
# INSERT STRUCTURE s22.c6h6_h2o_2
# INSERT PROPERTY s22.c6h6_h2o
# INSERT STRUCTURE s22.c6h6_nh3
# INSERT STRUCTURE s22.c6h6_nh3_1
# INSERT STRUCTURE s22.c6h6_nh3_2
# INSERT PROPERTY s22.c6h6_nh3
# INSERT STRUCTURE s22.c6h6_hcn
# INSERT STRUCTURE s22.c6h6_hcn_1
# INSERT STRUCTURE s22.c6h6_hcn_2
# INSERT PROPERTY s22.c6h6_hcn
# INSERT STRUCTURE s22.c6h6_c6h6_t
# INSERT STRUCTURE s22.c6h6_c6h6_t_1
# INSERT STRUCTURE s22.c6h6_c6h6_t_2
# INSERT PROPERTY s22.c6h6_c6h6_t
# INSERT STRUCTURE s22.indole_c6h6_t
# INSERT STRUCTURE s22.indole_c6h6_t_1
# INSERT STRUCTURE s22.indole_c6h6_t_2
# INSERT PROPERTY s22.indole_c6h6_t
# INSERT STRUCTURE s22.phenol_phenol
# INSERT STRUCTURE s22.phenol_phenol_1
# INSERT STRUCTURE s22.phenol_phenol_2
# INSERT PROPERTY s22.phenol_phenol

%% insert property ediff.
* INSERT: insert data into the database (PROPERTY)
# INSERT PROPERTY ediff.s22.nh3_nh3
# INSERT PROPERTY ediff.s22.nh3_nh3_1
# INSERT PROPERTY ediff.s22.nh3_nh3_2
# INSERT PROPERTY ediff.s22.h2o_h2o
# INSERT PROPERTY ediff.s22.h2o_h2o_1
# INSERT PROPERTY ediff.s22.h2o_h2o_2
# INSERT PROPERTY ediff.s22.h2co2_h2co2
# INSERT PROPERTY ediff.s22.h2co2_h2co2_1
# INSERT PROPERTY ediff.s22.h2co2_h2co2_2
# INSERT PROPERTY ediff.s22.formamide_formamide
# INSERT PROPERTY ediff.s22.formamide_formamide_1
# INSERT PROPERTY ediff.s22.formamide_formamide_2
# INSERT PROPERTY ediff.s22.uracil_uracil_hb
# INSERT PROPERTY ediff.s22.uracil_uracil_hb_1
# INSERT PROPERTY ediff.s22.uracil_uracil_hb_2
# INSERT PROPERTY ediff.s22.pyridoxine_aminopyridine
# INSERT PROPERTY ediff.s22.pyridoxine_aminopyridine_1
# INSERT PROPERTY ediff.s22.pyridoxine_aminopyridine_2
# INSERT PROPERTY ediff.s22.adenine_thymine_wcc1
# INSERT PROPERTY ediff.s22.adenine_thymine_wcc1_1
# INSERT PROPERTY ediff.s22.adenine_thymine_wcc1_2
# INSERT PROPERTY ediff.s22.ch4_ch4
# INSERT PROPERTY ediff.s22.ch4_ch4_1
# INSERT PROPERTY ediff.s22.ch4_ch4_2
# INSERT PROPERTY ediff.s22.c2h4_c2h4
# INSERT PROPERTY ediff.s22.c2h4_c2h4_1
# INSERT PROPERTY ediff.s22.c2h4_c2h4_2
# INSERT PROPERTY ediff.s22.c6h6_ch4
# INSERT PROPERTY ediff.s22.c6h6_ch4_1
# INSERT PROPERTY ediff.s22.c6h6_ch4_2
# INSERT PROPERTY ediff.s22.c6h6_c6h6_pd
# INSERT PROPERTY ediff.s22.c6h6_c6h6_pd_1
# INSERT PROPERTY ediff.s22.c6h6_c6h6_pd_2
# INSERT PROPERTY ediff.s22.pyrazine_pyrazine
# INSERT PROPERTY ediff.s22.pyrazine_pyrazine_1
# INSERT PROPERTY ediff.s22.pyrazine_pyrazine_2
# INSERT PROPERTY ediff.s22.uracil_uracil_stack
# INSERT PROPERTY ediff.s22.uracil_uracil_stack_1
# INSERT PROPERTY ediff.s22.uracil_uracil_stack_2
# INSERT PROPERTY ediff.s22.indole_c6h6_stack
# INSERT PROPERTY ediff.s22.indole_c6h6_stack_1
# INSERT PROPERTY ediff.s22.indole_c6h6_stack_2
# INSERT PROPERTY ediff.s22.adenine_thymine_stack
# INSERT PROPERTY ediff.s22.adenine_thymine_stack_1
# INSERT PROPERTY ediff.s22.adenine_thymine_stack_2
# INSERT PROPERTY ediff.s22.c2h4_c2h2
# INSERT PROPERTY ediff.s22.c2h4_c2h2_1
# INSERT PROPERTY ediff.s22.c2h4_c2h2_2
# INSERT PROPERTY ediff.s22.c6h6_h2o
# INSERT PROPERTY ediff.s22.c6h6_h2o_1
# INSERT PROPERTY ediff.s22.c6h6_h2o_2
# INSERT PROPERTY ediff.s22.c6h6_nh3
# INSERT PROPERTY ediff.s22.c6h6_nh3_1
# INSERT PROPERTY ediff.s22.c6h6_nh3_2
# INSERT PROPERTY ediff.s22.c6h6_hcn
# INSERT PROPERTY ediff.s22.c6h6_hcn_1
# INSERT PROPERTY ediff.s22.c6h6_hcn_2
# INSERT PROPERTY ediff.s22.c6h6_c6h6_t
# INSERT PROPERTY ediff.s22.c6h6_c6h6_t_1
# INSERT PROPERTY ediff.s22.c6h6_c6h6_t_2
# INSERT PROPERTY ediff.s22.indole_c6h6_t
# INSERT PROPERTY ediff.s22.indole_c6h6_t_1
# INSERT PROPERTY ediff.s22.indole_c6h6_t_2
# INSERT PROPERTY ediff.s22.phenol_phenol
# INSERT PROPERTY ediff.s22.phenol_phenol_1
# INSERT PROPERTY ediff.s22.phenol_phenol_2

%% insert calc
* INSERT: insert data into the database (CALC)
# INSERT EVALUATION (method=test;property=88;nvalue=1)
# INSERT EVALUATION (method=test;property=87;nvalue=1)
# INSERT EVALUATION (method=test;property=86;nvalue=1)
# INSERT EVALUATION (method=test;property=85;nvalue=1)
# INSERT EVALUATION (method=test;property=84;nvalue=1)
# INSERT EVALUATION (method=test;property=83;nvalue=1)
# INSERT EVALUATION (method=test;property=82;nvalue=1)
# INSERT EVALUATION (method=test;property=51;nvalue=1)
# INSERT EVALUATION (method=test;property=50;nvalue=1)
# INSERT EVALUATION (method=test;property=49;nvalue=1)
# INSERT EVALUATION (method=test;property=48;nvalue=1)
# INSERT EVALUATION (method=test;property=47;nvalue=1)
# INSERT EVALUATION (method=test;property=46;nvalue=1)
# INSERT EVALUATION (method=test;property=45;nvalue=1)
# INSERT EVALUATION (method=test;property=44;nvalue=1)
# INSERT EVALUATION (method=test;property=43;nvalue=1)
# INSERT EVALUATION (method=test;property=42;nvalue=1)
# INSERT EVALUATION (method=test;property=41;nvalue=1)
# INSERT EVALUATION (method=test;property=40;nvalue=1)
# INSERT EVALUATION (method=test;property=39;nvalue=1)
# INSERT EVALUATION (method=test;property=38;nvalue=1)
# INSERT EVALUATION (method=test;property=37;nvalue=1)
# INSERT EVALUATION (method=test;property=36;nvalue=1)
# INSERT EVALUATION (method=test;property=23;nvalue=1)
# INSERT EVALUATION (method=test;property=24;nvalue=1)
# INSERT EVALUATION (method=test;property=25;nvalue=1)
# INSERT EVALUATION (method=test;property=26;nvalue=1)
# INSERT EVALUATION (method=test;property=27;nvalue=1)
# INSERT EVALUATION (method=test;property=28;nvalue=1)
# INSERT EVALUATION (method=test;property=29;nvalue=1)
# INSERT EVALUATION (method=test;property=30;nvalue=1)
# INSERT EVALUATION (method=test;property=31;nvalue=1)
# INSERT EVALUATION (method=test;property=32;nvalue=1)
# INSERT EVALUATION (method=test;property=33;nvalue=1)
# INSERT EVALUATION (method=test;property=34;nvalue=1)
# INSERT EVALUATION (method=test;property=35;nvalue=1)
# INSERT EVALUATION (method=test;property=52;nvalue=1)
# INSERT EVALUATION (method=test;property=53;nvalue=1)
# INSERT EVALUATION (method=test;property=54;nvalue=1)
# INSERT EVALUATION (method=test;property=55;nvalue=1)
# INSERT EVALUATION (method=test;property=56;nvalue=1)
# INSERT EVALUATION (method=test;property=57;nvalue=1)
# INSERT EVALUATION (method=test;property=58;nvalue=1)
# INSERT EVALUATION (method=test;property=59;nvalue=1)
# INSERT EVALUATION (method=test;property=60;nvalue=1)
# INSERT EVALUATION (method=test;property=61;nvalue=1)
# INSERT EVALUATION (method=test;property=62;nvalue=1)
# INSERT EVALUATION (method=test;property=63;nvalue=1)
# INSERT EVALUATION (method=test;property=64;nvalue=1)
# INSERT EVALUATION (method=test;property=65;nvalue=1)
# INSERT EVALUATION (method=test;property=66;nvalue=1)
# INSERT EVALUATION (method=test;property=67;nvalue=1)
# INSERT EVALUATION (method=test;property=68;nvalue=1)
# INSERT EVALUATION (method=test;property=69;nvalue=1)
# INSERT EVALUATION (method=test;property=70;nvalue=1)
# INSERT EVALUATION (method=test;property=71;nvalue=1)
# INSERT EVALUATION (method=test;property=72;nvalue=1)
# INSERT EVALUATION (method=test;property=73;nvalue=1)
# INSERT EVALUATION (method=test;property=74;nvalue=1)
# INSERT EVALUATION (method=test;property=75;nvalue=1)
# INSERT EVALUATION (method=test;property=76;nvalue=1)
# INSERT EVALUATION (method=test;property=77;nvalue=1)
# INSERT EVALUATION (method=test;property=78;nvalue=1)
# INSERT EVALUATION (method=test;property=79;nvalue=1)
# INSERT EVALUATION (method=test;property=80;nvalue=1)
# INSERT EVALUATION (method=test;property=81;nvalue=1)

%% calc_ediff
* CALC_EDIFF: calculate and insert energy differences from total energies 
# INSERT EVALUATION (method=1;property=1;de=-1.57757)
# INSERT EVALUATION (method=1;property=2;de=-5.53069)
# INSERT EVALUATION (method=1;property=3;de=-18.7167)
# INSERT EVALUATION (method=1;property=4;de=-15.4892)
# INSERT EVALUATION (method=1;property=5;de=-20.9603)
# INSERT EVALUATION (method=1;property=6;de=-16.9635)
# INSERT EVALUATION (method=1;property=7;de=-15.6939)
# INSERT EVALUATION (method=1;property=8;de=0.223241)
# INSERT EVALUATION (method=1;property=9;de=0.711811)
# INSERT EVALUATION (method=1;property=10;de=0.487645)
# INSERT EVALUATION (method=1;property=11;de=1.6826)
# INSERT EVALUATION (method=1;property=12;de=1.06634)
# INSERT EVALUATION (method=1;property=13;de=-0.280557)
# INSERT EVALUATION (method=1;property=14;de=2.55719)
# INSERT EVALUATION (method=1;property=15;de=1.09468)
# INSERT EVALUATION (method=1;property=16;de=-0.414352)
# INSERT EVALUATION (method=1;property=17;de=-0.487164)
# INSERT EVALUATION (method=1;property=18;de=-0.137686)
# INSERT EVALUATION (method=1;property=19;de=-0.197283)
# INSERT EVALUATION (method=1;property=20;de=1.02501)
# INSERT EVALUATION (method=1;property=21;de=0.0430402)
# INSERT EVALUATION (method=1;property=22;de=-4.2815)

%% print evaluation
* PRINT: print the contents of the database 

| methodid| propid| #values| 1st value|
| 1| 88| 1| -301.7312791|
| 1| 87| 1| -301.7309249|
| 1| 86| 1| -603.469027|
| 1| 85| 1| -227.8908691|
| 1| 84| 1| -357.0284448|
| 1| 83| 1| -584.9192453|
| 1| 82| 1| -227.8909697|
| 1| 51| 1| -227.8910067|
| 1| 50| 1| -267.6170625|
| 1| 49| 1| -77.07260702|
| 1| 48| 1| -77.07260702|
| 1| 47| 1| -154.1440797|
| 1| 46| 1| -39.72674318|
| 1| 45| 1| -39.72674318|
| 1| 44| 1| -79.4531306|
| 1| 43| 1| -445.6595048|
| 1| 42| 1| -458.6127899|
| 1| 41| 1| -904.2973046|
| 1| 40| 1| -297.9593115|
| 1| 39| 1| -317.4563951|
| 1| 38| 1| -615.4427396|
| 1| 37| 1| -407.0748602|
| 1| 36| 1| -407.0748602|
| 1| 23| 1| -110.9108988|
| 1| 24| 1| -55.45419237|
| 1| 25| 1| -55.45419237|
| 1| 26| 1| -149.935376|
| 1| 27| 1| -74.96340216|
| 1| 28| 1| -74.96316009|
| 1| 29| 1| -372.4532966|
| 1| 30| 1| -186.2117348|
| 1| 31| 1| -186.2117348|
| 1| 32| 1| -333.3915592|
| 1| 33| 1| -166.6834378|
| 1| 34| 1| -166.6834378|
| 1| 35| 1| -814.1831228|
| 1| 52| 1| -39.72683291|
| 1| 53| 1| -455.7793141|
| 1| 54| 1| -227.8909977|
| 1| 55| 1| -227.8909977|
| 1| 56| 1| -518.7623837|
| 1| 57| 1| -259.3820341|
| 1| 58| 1| -259.382049|
| 1| 59| 1| -814.1540238|
| 1| 60| 1| -407.0767884|
| 1| 61| 1| -407.0767884|
| 1| 62| 1| -584.9152666|
| 1| 63| 1| -357.0283886|
| 1| 64| 1| -227.8909532|
| 1| 65| 1| -904.2739129|
| 1| 66| 1| -458.614619|
| 1| 67| 1| -445.6610385|
| 1| 68| 1| -152.9253127|
| 1| 69| 1| -77.07257386|
| 1| 70| 1| -75.85207857|
| 1| 71| 1| -302.8555041|
| 1| 72| 1| -227.8909365|
| 1| 73| 1| -74.96379126|
| 1| 74| 1| -283.3454498|
| 1| 75| 1| -227.8909722|
| 1| 76| 1| -55.45425822|
| 1| 77| 1| -319.5658018|
| 1| 78| 1| -227.8909015|
| 1| 79| 1| -91.67458591|
| 1| 80| 1| -455.7803182|
| 1| 81| 1| -227.890982|
| 1| 1| 1| -1.577567342|
| 1| 2| 1| -5.530691762|
| 1| 3| 1| -18.71667853|
| 1| 4| 1| -15.48920468|
| 1| 5| 1| -20.96029597|
| 1| 6| 1| -16.96346915|
| 1| 7| 1| -15.69391646|
| 1| 8| 1| 0.2232408238|
| 1| 9| 1| 0.7118111002|
| 1| 10| 1| 0.4876450765|
| 1| 11| 1| 1.682595735|
| 1| 12| 1| 1.066338138|
| 1| 13| 1| -0.280556974|
| 1| 14| 1| 2.557186432|
| 1| 15| 1| 1.094683995|
| 1| 16| 1| -0.4143519704|
| 1| 17| 1| -0.4871639022|
| 1| 18| 1| -0.1376856806|
| 1| 19| 1| -0.197282765|
| 1| 20| 1| 1.025013501|
| 1| 21| 1| 0.04304024705|
| 1| 22| 1| -4.281499624|

